exec/src/main/java/com/optum/sourcehawk/exec/FlattenCommand.java

```diff
@@ -0,0 +1,74 @@
+package com.optum.sourcehawk.exec;
+
+import com.optum.sourcehawk.core.scan.FlattenResult;
+import lombok.extern.slf4j.Slf4j;
+import lombok.val;
+import picocli.CommandLine;
+
+/**
+ * CLI entry point for executing Sourcehawk flatten command
+ *
+ * @author Christian Oestreich
+ */
+@Slf4j
+@CommandLine.Command(
+        name = FlattenCommand.COMMAND_NAME,
+        aliases = {"flat", "merge", "converge"},
+        description = "Flatten the sourcehawk.yml onto system by recursively reading and merging local and remote configurations"
+)
+class FlattenCommand extends AbstractCommand {
+
+    static final String COMMAND_NAME = "flatten";
+
+
+    @CommandLine.Option(
+            names = {"-o", "--output"},
+            description = "Override the default output name and path flattened file",
+            defaultValue = "sourcehawk-flattened.yml",
+            showDefaultValue = CommandLine.Help.Visibility.ALWAYS
+    )
+    private String output;
+
+    @CommandLine.Option(
+            names = {"-d", "--dry-run"},
+            description = "Display output to console instead of file system"
+    )
+    private boolean dryRun;
+
+    /**
+     * Bootstrap the command
+     *
+     * @param args the command line args
+     */
+    public static void main(final String... args) {
+        val command = new FlattenCommand();
+        val status = new CommandLine(command)
+                .setCaseInsensitiveEnumValuesAllowed(true)
+                .setTrimQuotes(true)
+                .execute(args);
+        Runtime.getRuntime()
+                .halt(status);
+    }
+
+    /**
+     * flatten the configuration
+     *
+     * @return the exit code
+     */
+    @Override
+    public Integer call() {
+        FlattenResult flattenResult;
+        val execOptions = buildExecOptions();
+        try {
```

Comment:
You can get some more code coverage here if you change this similar to recent updates I made in other commands.

Reply:
not sure what you mean here specifically, but coverage is pretty high